pkg/util/async/bounded_frequency_runner.go

```diff
@@ -189,13 +189,13 @@ func construct(name string, fn func(), minInterval, maxInterval time.Duration, b
 // Loop handles the periodic timer and run requests.  This is expected to be
 // called as a goroutine.
 func (bfr *BoundedFrequencyRunner) Loop(stop <-chan struct{}) {
-	klog.V(3).Infof("%s Loop running", bfr.name)
+	klog.V(3).InfoS("BoundedFrequencyRunner Loop running", "nameOfRunner", bfr.name)
```

Comment:
nameOfRunner->runner

Reply:
`runnerName` would be more appropriate imo based on the guidance: https://github.com/kubernetes/community/blob/master/contributors/devel/sig-instrumentation/migration-to-structured-logging.md#name-arguments